### ISSUE TITLE
Drupal sticky headers scroll sync experiment

### ISSUE THREAD

**Issue by pyrello** [uiowa/uiowa#3576]
Create an experimental implementation with using Drupal sticky headers to create responsive sticky headers that scroll vertically and horizontally for basic tables only. **This does not need to account for sticky first column or tables with `colspan`s.**

**Comment by bspeare** [comment]
Here's some instructions on how to use the Drupal library: https://github.com/uiowa/uiowa/issues/3402

**Comment by joewhitsitt** [comment]
>The idea with this is that when someone adds the Drupal sticky header class, we will automagically also load the scroll sync library and put in the correct classes to make it work. A bonus would be if there is a configuration file we can set up for the scroll sync library that let’s us modify which class is used to make it be the same as the sticky header class! Existing Drupal sticky header works fine for vertical scrolling, but does not handle horizontal scrolling. Scroll sync allows for syncing the horizontal scrolling of the sticky header and table.

I feel like an outsider looking in on this issue, but from what I have seen and then talked about with @GaryRidgway, I don't know how much the footprint would be reduced by this approach compared to what was removed from UIDS.

**Comment by bspeare** [comment]
Here's a proof of concept for sticky headers in a responsive table using only CSS: https://uids.brand.uiowa.edu/branches/feature_sticky_table_headers/components/preview/tables--responsive.html.    It looks like a recent update to Chromium has made this feature widely available: https://www.bram.us/2021/06/21/tablesng-improvements-to-table-rendering-in-chromium/#sticky-headers.   Please look at the UIDS link above and let me know if you have any objections to moving past this experiment and moving forward to using the CSS only solution?

**Comment by pyrello** [comment]
It looks good to me.

**Comment by dnkemper** [comment]
What @pyrello said!

**Comment by joewhitsitt** [comment]
POC looks done to me minus this border effect on scroll up. Seems to work in my non-Chromium based browsers too (Safari and Firefox).

<img width="332" alt="Screen Shot 2021-06-29 at 2 12 14 PM" src="https://user-images.githubusercontent.com/4663676/123854308-292ead00-d8e4-11eb-9e00-5a71c453e8f9.png">

**Comment by bspeare** [comment]
Here's the UIDS pr: https://github.com/uiowa/uids/pull/584.

**Comment by bspeare** [comment]
Closing this since the work is happening in https://github.com/uiowa/uids/pull/584.   Once the work is done in UIDS we should create a follow-up issue in `uiowa` to plan out the work to implement the UIDS solution.
